spec.md

```diff
@@ -173,6 +173,85 @@ minimally trusted storage servers is required.
 
 ## Authorization
 
+The ability to do cross-domain, operator-independent, standardized cloud storage
+operations requires an authorization system that is:
+
+* Modular and layered (for future agility / upgradability)
+* Not limited to traditional domain-based "usernames and passwords"
+* A hybrid, using object capability principles at its baseline, but also able
+  to provide ACL or RBAC-like functionality for user convenience. In other words,
+  the system needs to support both "anyone with the link can..." and "these are
+  specific people and groups allowed to..." styles of access control
+* Compatible with cross-domain replication
+* Compatible with end-to-end client side encryption (but also not rely on
+  encryption as the sole authorization method)
+* "Private by default".
+  That is, by default, unless otherwise specified, only the controller of a
+  space (or of a collection or resource) is authorized to perform any operation
+  (read, write, delete, etc)
+
+As the state of the art in cross-domain authorization advances, we expect there
+to be multiple profiles and specs that could be used to perform WAS API
+calls. However, to start with, this specification will focus on a single minimal
+authorization profile.
+
+### WAS Authorization Profile v0.1
+
+The initial W.A.S. Authorization Profile uses the following specifications.
+
+1. Identity (for controllers or clients/agents): [DID 1.1](https://www.w3.org/TR/did-1.1/)
+   or [CID 1.0](https://www.w3.org/TR/cid-1.0/) identifiers.
+2. Capability data model: [Authorization Capabilities for Linked Data v0.3](https://w3c-ccg.github.io/zcap-spec/)
+3. Protocol for obtaining authorization: Out of scope (implementers are encouraged
+   to use VC-API, OpenId4VP, or GNAP, as appropriate)
+4. Proof of Possession / authorization invocation: [HTTP Signatures (Cavage
+   draft 12)](https://datatracker.ietf.org/doc/html/draft-cavage-http-signatures)
+5. Access Control / Policy language data model: TBD
+
+#### Space `controller` and the Root of Trust
+
+Space `controller`s MUST be in the form of a [DID](https://www.w3.org/TR/did-1.1/)
+or a [CID](https://www.w3.org/TR/cid-1.0/) identifier.
```

Comment:
How is this enforced? If it is a DID, there is a syntax in did-core for that.
Syntax for a CID 1.0 appears to basically be like 'any uri' or 'string' so dilutes the requirement.

Reply:
Haha yeah that IS one of the problem with the CID spec :)

Reply:
Basically, it's enforced by "the client has to resolve the CID, and see if the resulting JSON is of this form"

Reply:
Ok. This CID thing is a pretty huge change in scope... could we just support did:key for now and add 'every https uri' in a PR where we also try to implement it? It begs many questions about acceptable latency for a verifier to be aware of a did doc rotation (unlike did:key), how to configure that, etc. The responsibility to resolve any/all https URIs just to validate a request is significant new expense in cost of operation, changes performance characteristics, etc.

Reply:
That said, I don't actually plan to implement CIDs soon, if ever. So, if you'd prefer, happy to take it out.

Reply:
Removed references to CID.